src/pages/signin/SignInPageLayout/SignInPageLayoutNarrow.js

```diff
@@ -1,97 +1,59 @@
 import React from 'react';
-import {
-    Image,
-    ScrollView, Text, View,
-} from 'react-native';
+import {ScrollView, Text, View} from 'react-native';
 import PropTypes from 'prop-types';
 import styles from '../../../styles/styles';
 import variables from '../../../styles/variables';
 import ExpensifyCashLogo from '../../../components/ExpensifyCashLogo';
-import welcomeScreenshot from '../../../../assets/images/welcome-screenshot.png';
 import TermsAndLicenses from '../TermsAndLicenses';
-import WelcomeText from '../../../components/WelcomeText';
-import openURLInNewTab from '../../../libs/openURLInNewTab/index.native';
-import CONST from '../../../CONST';
 import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
 
 const propTypes = {
-
     /** The children to show inside the layout */
     children: PropTypes.node.isRequired,
 
     /** Whether we should show the welcome elements */
-    shouldShowWelcomeText: PropTypes.bool,
-    shouldShowWelcomeScreenshot: PropTypes.bool,
+    // shouldShowWelcomeText: PropTypes.bool,
+    // shouldShowWelcomeScreenshot: PropTypes.bool,
```

Comment:
Is there a reason these are commented out? Please either remove them if they are not needed, or uncomment them. Same for the other instances in this diff.

Reply:
I am some queries to be resolved 
https://github.com/Expensify/Expensify.cash/issues/2938#issuecomment-847191266

Reply:
👋 @pranshuchittora were your queries resolved here? Please let us know if not.

Reply:
Changes done.
I have removed the props :)